src/ncp/ncp_base.cpp

```diff
@@ -748,6 +757,99 @@ void NcpBase::HandleEnergyScanResult(otEnergyScanResult *aResult)
 }
 
 // ----------------------------------------------------------------------------
+// MARK: Raw Link-Layer Datapath Glue
+// ----------------------------------------------------------------------------
+
+#if OPENTHREAD_ENABLE_RAW_LINK_API
+
+void NcpBase::LinkRawReceiveDone(otInstance *, RadioPacket *aPacket, ThreadError aError)
+{
+    sNcpContext->LinkRawReceiveDone(aPacket, aError);
+}
+
+void NcpBase::LinkRawReceiveDone(RadioPacket *aPacket, ThreadError aError)
+{
+    ThreadError errorCode = kThreadError_None;
+    uint16_t flags = 0;
+
+    SuccessOrExit(errorCode = OutboundFrameBegin());
+
+    if (aPacket->mDidTX)
+    {
+        flags |= SPINEL_MD_FLAG_TX;
+    }
+
+    // Append frame header and frame length
+    SuccessOrExit(
+        errorCode = OutboundFrameFeedPacked(
+            "CiiS",
+            SPINEL_HEADER_FLAG | SPINEL_HEADER_IID_0,
+            SPINEL_CMD_PROP_VALUE_IS,
+            SPINEL_PROP_STREAM_RAW,
+            (aError == kThreadError_None) ? aPacket->mLength : 0
+        )
+    );
+
+    if (aError == kThreadError_None)
+    {
+        // Append the frame contents
+        SuccessOrExit(
+            errorCode = OutboundFrameFeedData(
+                aPacket->mPsdu,
+                aPacket->mLength
+            )
+        );
+    }
+
+    // Append metadata (rssi, etc)
+    SuccessOrExit(
+        errorCode = OutboundFrameFeedPacked(
+            "ccSiCC",
+            aPacket->mPower,    // TX Power
+            -128,               // Noise Floor (Currently unused)
```

Comment:
This is copied from elsewhere in NCP and (as noted) is currently unused.  I wouldn't worry about it here.